sql/plan/set_op.go

```diff
@@ -0,0 +1,249 @@
+// Copyright 2020-2021 Dolthub, Inc.
```

Comment:
If you use `git mv`, you can make git see that this is a rename of `union.go`

Reply:
Just tried, it didn't do it

Reply:
might work if you do it inside a to-main squash